sql-native/src/main/java/org/ballerinalang/sql/nativeimpl/OutParameterProcessor.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Don't change the license year for file renames.  Check all the places.

Reply:
ok we will change it.

Reply:
Resolved with 43eb47e